### PR TITLE
Update version_notes.adoc with v0.13 versions on testnets

### PR DESCRIPTION
**Pull request by odednaor**
### Description of the Changes

Release notes of Starknet v0.13, and update versions tables on Starknet, Cairo, and Sierra version numbers.

### PR Preview URL

https://starknet-io.github.io/starknet-docs/pr-1023/documentation/starknet_versions/version_notes/

### Check List

- [x] Changes have been done against master branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1023)

### CODE REVIEW

**Comment by odednaor** [comment]
@stoobie What should we do with the v0.13 release notes on the [upcoming versions page](https://starknet-io.github.io/starknet-docs/pr-1023/documentation/starknet_versions/version_notes/) while mainnet is not yet upgraded to v0.13?

**Review by ArielElp — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1023)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @odednaor and @stoobie)

___
*[`components/Starknet/modules/starknet_versions/pages/version_notes.adoc` line 32 at r1](https://reviewable.io/reviews/starknet-io/starknet-docs/1023#-NlXwzm0C8iTasel9UKF:-NlXwzm0C8iTasel9UKG:b-vfgi4k) ([raw file](https://github.com/starknet-io/starknet-docs/blob/e73dc92987760c6fc043724a62632b6f1fa364d0/components/Starknet/modules/starknet_versions/pages/version_notes.adoc#L32)):*
> ```AsciiDoc
> ** Fee payment in STRK
> ** Reserved fields for future features, such as volition and payment master
> * `get_block` API: Rename the `gas_price` field to `eth_l1_gas_price` and add a new field `strk_l1_gas_price` (this applies to old blocks as well)
> ```

Replace this point with a link to v0.6.0 spec https://github.com/starkware-libs/starknet-specs/releases/tag/v0.6.0
___
*[`components/Starknet/modules/starknet_versions/pages/version_notes.adoc` line 33 at r1](https://reviewable.io/reviews/starknet-io/starknet-docs/1023#-NlYYTksA66UdssvT3PS:-NlYYTksA66UdssvT3PT:bbtzbi0) ([raw file](https://github.com/starknet-io/starknet-docs/blob/e73dc92987760c6fc043724a62632b6f1fa364d0/components/Starknet/modules/starknet_versions/pages/version_notes.adoc#L33)):*
> ```AsciiDoc
> ** Reserved fields for future features, such as volition and payment master
> * `get_block` API: Rename the `gas_price` field to `eth_l1_gas_price` and add a new field `strk_l1_gas_price` (this applies to old blocks as well)
> * Update Sierra version used in Starknet to `1.4.0` (introduced in crate version `v2.4.0` of the cairo package)
> ```

replace this with a link here https://community.starknet.io/t/cairo-v2-4-0-is-out/109275
___
*[`components/Starknet/modules/starknet_versions/pages/version_notes.adoc` line 35 at r1](https://reviewable.io/reviews/starknet-io/starknet-docs/1023#-NlYYeqVCSLJpqrNb9As:-NlYYeqW7Wlv8q-jI7Sy:b-owjsbz) ([raw file](https://github.com/starknet-io/starknet-docs/blob/e73dc92987760c6fc043724a62632b6f1fa364d0/components/Starknet/modules/starknet_versions/pages/version_notes.adoc#L35)):*
> ```AsciiDoc
> * Update Sierra version used in Starknet to `1.4.0` (introduced in crate version `v2.4.0` of the cairo package)
> * Improved performance of `secp256k1_mul` and `secp256r1_mul` syscalls
> ```

we need a another bullet point about cost reduction: computation cost is reduced by roughly 2x (cairo steps and builtins), DA cost is reduced by roughly 10%.

**Review by stoobie — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1023)* status: 0 of 1 files reviewed, 7 unresolved discussions (waiting on @ArielElp and @odednaor)

___
*[`components/Starknet/modules/starknet_versions/pages/version_notes.adoc` line 35 at r1](https://reviewable.io/reviews/starknet-io/starknet-docs/1023#-NlYYeqVCSLJpqrNb9As:-NlYfGXR9tP09BjIx0tJ:b-83330m) ([raw file](https://github.com/starknet-io/starknet-docs/blob/e73dc92987760c6fc043724a62632b6f1fa364d0/components/Starknet/modules/starknet_versions/pages/version_notes.adoc#L35)):*
<details><summary><i>Previously, ArielElp wrote…</i></summary><blockquote>

we need a another bullet point about cost reduction: computation cost is reduced by roughly 2x (cairo steps and builtins), DA cost is reduced by roughly 10%.
</blockquote></details>

PTAL.
___
*[`components/Starknet/modules/starknet_versions/pages/version_notes.adoc` line 32 at r2](https://reviewable.io/reviews/starknet-io/starknet-docs/1023#-NlYdMWeE7lUN8wTZztX:-NlYdMWeE7lUN8wTZztY:undefined) ([raw file](https://github.com/starknet-io/starknet-docs/blob/4ddaae955d2ba31b8dedc90838372cac485e495c/components/Starknet/modules/starknet_versions/pages/version_notes.adoc#L32)):*
> ```AsciiDoc
> ** Fee payment in STRK
> ** Reserved fields for future features, such as volition and payment master
> * `get_block` API: Rename the `gas_price` field to `eth_l1_gas_price` and add a new field `strk_l1_gas_price` (this applies to old blocks as well)
> ```





_Suggestion:_
```AsciiDoc
`get_block` API: The `gas_price` field is replaced by the `eth_l1_gas_price` and `strk_l1_gas_price` fields. This change applies also to existing blocks. For more information on the new fields, see the link:https://github.com/starkware-libs/starknet-specs/releases/tag/v0.6.0[JSON RPC API Spec on Github]
```
___
*[`components/Starknet/modules/starknet_versions/pages/version_notes.adoc` line 33 at r2](https://reviewable.io/reviews/starknet-io/starknet-docs/1023#-NlYe91SEox55DzHU_Q6:-NlYe91SEox55DzHU_Q7:undefined) ([raw file](https://github.com/starknet-io/starknet-docs/blob/4ddaae955d2ba31b8dedc90838372cac485e495c/components/Starknet/modules/starknet_versions/pages/version_notes.adoc#L33)):*
> ```AsciiDoc
> ** Reserved fields for future features, such as volition and payment master
> * `get_block` API: Rename the `gas_price` field to `eth_l1_gas_price` and add a new field `strk_l1_gas_price` (this applies to old blocks as well)
> * Update Sierra version used in Starknet to `1.4.0` (introduced in crate version `v2.4.0` of the cairo package)
> ```





_Suggestion:_
```AsciiDoc
* Sierra v1.4.0. This new version of Sierra is part of Crate v2.4.0, in the Cairo 2.4.0 package.
```
___
*[`components/Starknet/modules/starknet_versions/pages/version_notes.adoc` line 34 at r2](https://reviewable.io/reviews/starknet-io/starknet-docs/1023#-NlYebPP-729Bl_lX5FD:-NlYebPP-729Bl_lX5FE:b-ahv4n1) ([raw file](https://github.com/starknet-io/starknet-docs/blob/4ddaae955d2ba31b8dedc90838372cac485e495c/components/Starknet/modules/starknet_versions/pages/version_notes.adoc#L34)):*
> ```AsciiDoc
> * `get_block` API: Rename the `gas_price` field to `eth_l1_gas_price` and add a new field `strk_l1_gas_price` (this applies to old blocks as well)
> * Update Sierra version used in Starknet to `1.4.0` (introduced in crate version `v2.4.0` of the cairo package)
> * Improved performance of `secp256k1_mul` and `secp256r1_mul` syscalls
> ```

We don't document these syscalls. Should we?
___
*[`components/Starknet/modules/starknet_versions/pages/version_notes.adoc` line 34 at r2](https://reviewable.io/reviews/starknet-io/starknet-docs/1023#-NlYep5GC0r2sDXgy5Cn:-NlYep5GC0r2sDXgy5Co:undefined) ([raw file](https://github.com/starknet-io/starknet-docs/blob/4ddaae955d2ba31b8dedc90838372cac485e495c/components/Starknet/modules/starknet_versions/pages/version_notes.adoc#L34)):*
> ```AsciiDoc
> * `get_block` API: Rename the `gas_price` field to `eth_l1_gas_price` and add a new field `strk_l1_gas_price` (this applies to old blocks as well)
> * Update Sierra version used in Starknet to `1.4.0` (introduced in crate version `v2.4.0` of the cairo package)
> * Improved performance of `secp256k1_mul` and `secp256r1_mul` syscalls
> ```





_Suggestion:_
```AsciiDoc
* Improved performance of `secp256k1_mul` and `secp256r1_mul` syscalls
* Computation cost is reduced approximately 200% as a result of reduced Cairo steps and increased use of builtins. L1 data availability cost is reduced by approximtately 10%.
```

**Review by stoobie — CHANGES_REQUESTED** [review]
@odednaor I added some comments and suggestions in Reviewable.

**Comment by stoobie** [comment]
> @stoobie What should we do with the v0.13 release notes on the [upcoming versions page](https://starknet-io.github.io/starknet-docs/pr-1023/documentation/starknet_versions/version_notes/) while mainnet is not yet upgraded to v0.13?

Well, the info on that page is more detailed than the info in this PR. I thought to just move that info into the Release notes and remove that page for now. We clearly state the status on testnet and Mainnet.

File: components/Starknet/modules/starknet_versions/pages/version_notes.adoc [hunk 1]
```diff
@@ -16,10 +16,25 @@ include::ROOT:partial$snippet_important_goerli2_removed.adoc[]
...
 |Sepolia Testnet|0.13.0|1.4.0|2.0.0 - 2.4.0
 |Goerli Testnet (deprecated)|0.13.0|1.4.0|2.0.0 - 2.4.0
 |===
 
+[id="version0.13.0"]
+== Starknet v0.13.0 (December 13th 23)
+
+Starknet v0.13.0 is live on testnet.
+
+* Add transaction v3:
```
Comment (stoobie) [inline components/Starknet/modules/starknet_versions/pages/version_notes.adoc:29]:
```suggestion
Starknet 0.13.0 includes the following changes:

* v3 transactions, including:
```

File: components/Starknet/modules/starknet_versions/pages/version_notes.adoc [hunk 2]
```diff
@@ -16,10 +16,25 @@ include::ROOT:partial$snippet_important_goerli2_removed.adoc[]
...
+** Fee payment in STRK
+** Reserved fields for future features, such as volition and payment master
```
Comment (stoobie) [inline components/Starknet/modules/starknet_versions/pages/version_notes.adoc:31]:
```suggestion
** Reserved fields for future features, such as Volition and payment master
```

File: components/Starknet/modules/starknet_versions/pages/version_notes.adoc [hunk 3]
```diff
@@ -16,10 +16,27 @@ include::ROOT:partial$snippet_important_goerli2_removed.adoc[]
...
+
+Starknet 0.13.0 includes the following changes:
+
+* v3 transactions, including:
+** Fee payment in STRK
+** Reserved fields for future features, such as Volition and payment master
+* `get_block` API: The `gas_price` field is replaced by the `eth_l1_gas_price` and `strk_l1_gas_price` fields. This change applies also to existing blocks. For more information on the new fields, see the link:https://github.com/starkware-libs/starknet-specs/releases/tag/v0.6.0[JSON RPC API Spec on Github]
+* Sierra v1.4.0. This new version of Sierra is part of Crate v2.4.0, in the Cairo 2.4.0 package. For more information, see link:https://community.starknet.io/t/cairo-v2-4-0-is-out/109275[Cairo v2.4.0 is out!] on the Community forum.
+* Improved performance of `secp256k1_mul` and `secp256r1_mul` syscalls
+* Computation cost is reduced approximately 200% as a result of reduced Cairo steps and increased use of builtins. L1 data availability cost is reduced by approximtately 10%.
```
Comment (stoobie) [inline components/Starknet/modules/starknet_versions/pages/version_notes.adoc:37]:
```suggestion
* Computation cost is reduced by approximately 200% as a result of reduced Cairo steps and increased use of builtins. L1 data availability cost is reduced by approximtately 10%.
```

**Pull request merged**